src/frontends/onnx/frontend/src/ops_bridge.cpp

```diff
@@ -457,6 +458,7 @@ OperatorsBridge::OperatorsBridge() {
     REGISTER_OPERATOR("TopK", 10, topk);
     REGISTER_OPERATOR("TopK", 11, topk);
     REGISTER_OPERATOR("Transpose", 1, transpose);
+    REGISTER_OPERATOR("Trilu", 1, trilu);
```

Comment:
Looks strange that we register this operation as from opset-1.
Not good workaround.

Reply:
For now, this is and was a standard in approach for ONNX Frontend to register and make available new operations since onnx opset-1. Proper opset availability is crucial only for the next versions of the operation, if there are registered more than one. 
For example if ONNX introduce next Trilu version in onnx opset-18 we will have internally `set_1::Trilu` for onnx opset (1, 17 and `set_18::Trilu` for onnx opset (18, +) until the next version of the Trilu operation.

Of course we can discuss some improvements of this logic, but it is rather out of this PR scope.

Reply:
Registering to new ops to opset1 is kind of legacy approach we still follow. However, technically (except this UnsupportedVersion exception) I can't see any reason we couldn't register Trilu starting from opset 14 instead of opset 1, regardless if we could have next version of Trilu in e.g. opset 18 or not. But in my opinion, we can think about that later and keep this PR as is.